swimlane/core/resources/usergroup.py

```diff
@@ -79,14 +81,22 @@ class Group(UserGroup):
 
     Attributes:
         description (str): Group description
+        users (list): List of users belonging to group.
```

Comment:
Put the type as `GroupUsersCursor`. Know I said it didn't need it, but add a small section to the group example docs with brief details and example showing iteration over the group.users